cmsFormPage.php

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
I see lots of `<br>` tags!!  😱

You want to try and replace those for CSS

Reply:
Good point. I'll look at it in later code refactoring.

Reply:
+1